packages/upgrade/src/spawn.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright IBM Corp. 2019, 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import spawn from 'cross-spawn';
```

Comment:
We may be able to get away with `execa` now for promise support: https://www.npmjs.com/package/execa